build_manifest.sh

```diff
@@ -15,6 +15,7 @@
 PROJECTS=(
   circuits:circuits/cpp:./dockerfiles/Dockerfile.wasm-linux-clang:circuits-wasm-linux-clang
   l1-contracts:l1-contracts
+  noir-contracts:yarn-project
```

Comment:
was this meant to be included?

Reply:
Yes. It's now need to build the sandbox locally.